cmd/buildah/run.go

```diff
@@ -73,10 +73,15 @@ func runCmd(c *cli.Context) error {
 		return errors.Wrapf(err, "error reading build container %q", name)
 	}
 
+	runtimeFlags := []string{}
+	for _, arg := range c.StringSlice("runtime-flag") {
```

Comment:
Can we check the arg and add the '--' only if not present?  I.e. if a user wanted to do buildah run --runtime-flag '--debug' that should be OK me thinks.

Reply:
That will not work because the CLI tool would think of --debug as another flag and will complain that --runtime-flag does not have a value.

Reply:
Gotta love the CLI tool, OK scratch that thought then.

Reply:
yup, I added a note in the man page and fixed the examples there. So should be pretty obvious on how to work with the `--runtime-flag` flag.